examples/htool.c

```diff
@@ -251,6 +251,413 @@ static int do_target_reset(uint32_t reset_option) {
       sizeof(req), NULL, 0, NULL);
 }
 
+struct payload_status_response_header {
+  uint8_t version;
+  uint8_t lockdown_state;
+  uint8_t active_half;
+  uint8_t region_count;
+} __attribute__((packed));
+
+struct payload_region_state {
+  uint8_t validation_state; /* enum payload_validation_state */
+  uint8_t failure_reason;   /* enum payload_validation_failure_reason */
+  uint8_t reserved_0;
+  uint8_t image_type; /* enum image_type (dev, prod, breakout) */
+  uint16_t key_index;
+  uint16_t reserved_1;
+  uint32_t image_family; /* handy to disambiguate during enumeration */
+  uint32_t version_major;
+  uint32_t version_minor;
+  uint32_t version_point;
+  uint32_t version_subpoint;
+  uint32_t descriptor_offset; /* can be used to pull the image hash/signature */
+} __attribute__((packed));
+
+const char* SpsEepromLockdownStatusString(uint8_t st) {
+  switch (st) {
+    case 0:
```

Comment:
nit: It's probably wise to have actual machine-readable enums for these values, here and below -- but I won't insist.

Reply:
Removed payload-related structures and constants from this change for now.